taxcalc/policy_current_law.json

```diff
@@ -15643,6 +17821,10 @@
             },
             {
                 "year": 2022,
+                "value": 125
```

Comment:
@bodiyang What is the reason you've updated the age of EITC eligibility with your updates for inflation?  Also, why is 2020 listed after 2022?

Reply:
1. Before 2021 the upper age limit of EITC is 64; for the year 2021 the upper age limit of EITC is eliminated, while the tax instruction form of 2022 does not mention weather the upper age limit will be reimposed or not, so should it be set as no upper age limit as of 2021?
2. Yes, 2020 should be listed before 2021 & 2022, this needs to be revised

Reply:
@bodiyang It's not totally clear to me reading the text of [the ARPA](https://www.congress.gov/117/bills/hr1319/BILLS-117hr1319enr.pdf), but my understanding was that this was a temporary provision and the `policy_current_law.json` file had been set for this to expire after 2021 before you changed it. Unless you can find evidence otherwise, I think you should keep the age 64 in 2022.